### ISSUE TITLE
Cat image is missing alt text.

### ISSUE THREAD

**Issue by MassDebates** [denysvitali/thebestmotherfuckingwebsite#40]
"This website is even better than the last guy and the guy before him."

One of the reasons cited was Accessibility.

According to [support.google.com](https://support.google.com/webmasters/answer/114016?hl=en#descriptive-alt-text), 
"Alt text (text that describes an image) improves accessibility for people who can't see images on web pages, including users who use screen readers or have low-bandwidth connections."

"When choosing alt text, focus on creating useful, information-rich content that uses keywords appropriately and is in context of the content of the page. Avoid filling alt attributes with keywords (keyword stuffing) as it results in a negative user experience and may cause your site to be seen as spam."

Alt text is important, especially for accessibility. Here is a link to a video showing how critical alt text is for blind and visually-disabled visitors: https://www.youtube.com/watch?v=dEbl5jvLKGQ

Offending code's nearest containing div: 
`<div class="picture-left"><img src="static/e84d13374f0772372eaa86614800bd58.jpg" title="A nice and cute cat"><p class="caption">A cat, just to keep the context of the website.</p></div>`

**Comment by denysvitali** [comment]
Indeed, I've fixed the issue in my most recent commit. Feel free to comment or re-open the issue if you feel that the text is not descriptive enough / is still not accessible +1:
